neurodamus/node.py

```diff
@@ -1306,7 +1314,7 @@ def _sim_init_neuron(self):
             Nd.t = -1e9
             prev_dt = Nd.dt
             Nd.dt = fwd_skip * 0.1
-            for flushIndex in range(10):
+            for _flushIndex in range(10):
```

Comment:
this looks very weird to me; we set time to be negative? and then we advance it?

Reply:
I think it is on purpose. Objectives:
- reach steady-state giving some initial time to the simulation to settle
- have the time stamps aligned: we want to skip the initial settling period and start counting time (for the recordings) from our chosen 0 (where steady state is reached)

Solution:
- the simulation starts from a negative time step and proceeds until 0. Recording should be skipped?

I let @WeinaJi correct me if I recall incorrectly.

Reply:
Ok, I just found it strange; sounds like something that would be helpful to have a comment.
Also, are there particular meanings to -1e9, for instance, how can one tell if the settling will finish in that time?

Reply:
This will be fixed if/when `fwd_skip` is removed